core/storage/suggestion/gae_models.py

```diff
@@ -196,3 +200,59 @@ def get_all_stale_suggestions(cls):
                 0, 0, 0, THRESHOLD_TIME_BEFORE_ACCEPT_IN_MSECS))
         return cls.get_all().filter(cls.status == STATUS_IN_REVIEW).filter(
             cls.last_updated < threshold_time).fetch()
+
+    @classmethod
+    def get_all_score_categories(cls):
+        """Gets all the score categories for which suggestions have been
+        created.
+
+        Returns:
+            list(str). A list of all the score categories.
+        """
+        query_set = cls.query(projection=['score_category'], distinct=True)
+        return [data.score_category for data in query_set]
+
+
+class ReviewerRotationTrackingModel(base_models.BaseModel):
```

Comment:
when (and where) do you add entries to the list in this model?

Reply:
Same as https://github.com/oppia/oppia/pull/5387#discussion_r209459554